src/status_im2/common/scan_qr_code/view.cljs

```diff
@@ -0,0 +1,239 @@
+(ns status-im2.common.scan-qr-code.view
+  (:require [clojure.string :as string]
+            [reagent.core :as reagent]
+            [oops.core :as oops]            
+            [status-im2.common.scan-qr-code.style :as style]
```

Comment:
please don't import styles like this. If you're importing some component from another namespace then import the component directly. It seems in this file you have copy pasted a lot of code where you could have just moved it to somewhere common and reused it within this page.

This would be the better approach imo. If you have any doubts about where to put particular bits of common code then reach out to a dev for some feedback 👍

Reply:
I am importing the scanner related style file So this is why I have to import this file

Reply:
sure, but can't you just import the components you are using directly so that you don't need the styles as they are encapsulated by the component you pull in

Reply:
Apologies @teoarjun, I believe you are correct here. 👍

Reply:
Thanks